apollo-router/src/lib.rs

```diff
@@ -74,7 +74,7 @@ pub enum FederatedServerError {
 pub enum SchemaKind {
     /// A static schema.
     #[display(fmt = "Instance")]
-    Instance(graphql::Schema),
+    Instance(Box<graphql::Schema>),
```

Comment:
why is the Box required now?

Reply:
clippy big variant :disappointed:

Reply:
:see_no_evil:

Reply:
yeah and tbh I'm not sure which ones can be ignored. It's not like we instantiate those objects a lot or "372 bytes" is such a big deal :confounded: really in doubt here so I followed clippy's advice which might be wrong

Reply:
I am not a fan of adding a Box everywhere for this, and I know we can tell clippy to ignore that one, but I do not care too much about having it either way

Reply:
Same... I think that will be for a future refactoring of the state_machine.rs at some point